src/model/resources/storage/long_duration_storage.jl

```diff
@@ -86,14 +96,23 @@ function long_duration_storage!(EP::Model, inputs::Dict)
 	# Build up inventory can be positive or negative
 	@variable(EP, vdSOC[y in STOR_LONG_DURATION, w=1:REP_PERIOD])
 
+			
+	# State of charge held in reserve for storage at beginning of each modeled period n
+	@variable(EP, vCAPRES_socw[y in STOR_LONG_DURATION, n in MODELED_PERIODS_INDEX] >= 0)
+
+	# Build up in storage inventory held in reserve over each representative period w
+	# Build up inventory can be positive or negative
+	@variable(EP, vCAPRES_dsoc[y in STOR_LONG_DURATION, w=1:REP_PERIOD])
```

Comment:
Instead of setting these to zero later via a constraint if capacity_reserve_margin is not 1, could you avoid creating them in the first place?